src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -116,31 +128,33 @@ private CommandResult deleteByIndex(Model model) throws CommandException {
     }
 
     /**
-     * Deletes a employee by name
+     * Deletes an employee by name
      *
      * @param model the model to execute the command
      * @return the result of the command
      * @throws CommandException if the employee is not found
      */
-    CommandResult deleteByName(Model model) throws CommandException {
+    private CommandResult deleteByName(Model model) throws CommandException {
         List<Employee> lastShownList = model.getFilteredEmployeeList();
-        List<Employee> employeesWithTargetName = new ArrayList<>();
+        requireNonNull(lastShownList, "Employee list is null");
 
+        List<Employee> employeesWithTargetName = new ArrayList<>();
         for (Employee employee : lastShownList) {
             if (employee.getName().fullName.equalsIgnoreCase(targetName)) {
                 employeesWithTargetName.add(employee);
             }
         }
 
         if (employeesWithTargetName.size() > 1) {
-            throw new CommandException("Multiple employees with this name found. Please delete by uid.\n"
-                    + "Format: delete uid/UID_NUMBER\n" + "Example: delete uid/101");
-        } else if (employeesWithTargetName.size() == 1) {
-            model.deleteEmployee(employeesWithTargetName.get(0));
-            return new CommandResult(
-                    String.format(MESSAGE_DELETE_EMPLOYEE_SUCCESS, Messages.format(employeesWithTargetName.get(0))));
-        } else {
+            LOGGER.log(Level.WARNING, "Multiple employees found with name: " + targetName);
+            throw new CommandException(MESSAGE_MULTIPLE_EMPLOYEES_FOUND);
```

Comment:
good use of a constant instead of just typing out the message, allows for easier changes in the future.